html/css/style.css

```diff
@@ -41,12 +54,66 @@
 .secondrow__first {
     padding: 1%;
     flex-basis: 60%;
+    display: flex;
+    flex-direction: column;
 }
 
 .secondrow__second {
     padding: 1%;
     flex-basis: 40%;
     background-color: #efeef0;
+    display: flex;
+    flex-direction: column;
+}
+
+.first__wallet {
+    padding-top: 1%;
+    padding-bottom: 1%;
```

Comment:
can use only padding instead of these both and give 0 for left and right

Reply:
Done